src/stories/vocabulary_icons.stories.mdx

```diff
@@ -85,64 +77,29 @@ smallest possible file size bundle.
   </IconItem>
 </IconGallery>
 
-### Internet
+### Hieroglyphs
 
 <IconGallery>
-  <IconItem name="search">
-    <i class="icon search"></i>
+  <IconItem name="globe">
+   <i class="icon globe"></i>
   </IconItem>
   <IconItem name="info">
     <i class="icon info"></i>
   </IconItem>
-  <IconItem name="globe">
-   <i class="icon globe"></i>
-  </IconItem>
   <IconItem name="external-link">
     <i class="icon external-link"></i>
   </IconItem>
-</IconGallery>
-
-## Social media
-
-### Monochromatic
-
-<IconGallery>
-  <IconItem name="instagram">
-    <i class="icon instagram"></i>
-  </IconItem>
-  <IconItem name="twitter">
-    <i class="icon twitter"></i>
-  </IconItem>
-  <IconItem name="linkedin">
-    <i class="icon linkedin"></i>
-  </IconItem>
-  <IconItem name="facebook">
-    <i class="icon facebook"></i>
-  </IconItem>
-  <IconItem name="pinterest">
-    <i class="icon pinterest"></i>
-  </IconItem>
-</IconGallery>
-
-### Colored
-
-Using the `.colored` class, you can use a colored version of these icons.
-
-<IconGallery>
-  <IconItem name="instagram">
-    <i class="icon instagram colored"></i>
-  </IconItem>
-  <IconItem name="twitter">
-    <i class="icon twitter colored"></i>
+  <IconItem name="search">
+    <i class="icon search"></i>
   </IconItem>
-  <IconItem name="linkedin">
-    <i class="icon linkedin colored"></i>
+  <IconItem name="sort">
+    <i class="icon sort"></i>
   </IconItem>
-  <IconItem name="facebook">
-    <i class="icon facebook colored"></i>
+  <IconItem name="filter">
```

Comment:
The `filters` and `sliders` icons need a colored version too, because on CC Search we toggle their colors to indicate that a filter is activated

Reply:
You can color any icon with CSS `color` property. They're basically text.

Reply:
I only provided colored versions for the brands because their colors are very specific and not present in Vocabulary color tokens.

Reply:
@brenoferreira is there a particular color that is not present in Vocabulary?